Tests/Resources/converted-event.json

```diff
@@ -1,4975 +1,3836 @@
-            "uuid": "51D6C3AB-9702-3564-AED4-483053FA0E5E",
-            "type": "apple",
-            "minor_version": 2,
-            "revision_version": 67
-        }, {
-            "major_version": 1,
-            "image_size": 823296,
-            "name": "\/System\/Library\/PrivateFrameworks\/CoreUtils.framework\/CoreUtils",
-            "cpu_subtype": 0,
-            "image_vmaddr": "0x18e451000",
-            "image_addr": "0x198061000",
-            "cpu_type": 16777228,
-            "uuid": "C9E7F832-9190-3BAC-93ED-C3E87D21EF14",
-            "type": "apple",
-            "minor_version": 0,
-            "revision_version": 0
-        }, {
-            "major_version": 1,
-            "image_size": 45056,
-            "name": "\/System\/Library\/PrivateFrameworks\/BluetoothManager.framework\/BluetoothManager",
-            "cpu_subtype": 0,
-            "image_vmaddr": "0x18a776000",
-            "image_addr": "0x194386000",
-            "cpu_type": 16777228,
-            "uuid": "6D7F28A1-EE64-30E8-ABF3-405997CE4969",
-            "type": "apple",
-            "minor_version": 0,
-            "revision_version": 0
-        }, {
-            "major_version": 1,
-            "image_size": 57344,
-            "name": "\/System\/Library\/PrivateFrameworks\/MobileBluetooth.framework\/MobileBluetooth",
-            "cpu_subtype": 0,
-            "image_vmaddr": "0x18a72d000",
-            "image_addr": "0x19433d000",
-            "cpu_type": 16777228,
-            "uuid": "F2526AC2-1040-369D-AD9D-B947D1C66CAD",
-            "type": "apple",
-            "minor_version": 0,
-            "revision_version": 0
-        }, {
-            "major_version": 1,
-            "image_size": 180224,
-            "name": "\/System\/Library\/PrivateFrameworks\/IMSharedUtilities.framework\/IMSharedUtilities",
-            "cpu_subtype": 0,
-            "image_vmaddr": "0x19bc3d000",
-            "image_addr": "0x1a584d000",
-            "cpu_type": 16777228,
-            "uuid": "82638556-97BB-3ED6-820E-E5357619F000",
-            "type": "apple",
-            "minor_version": 0,
-            "revision_version": 0
-        }]
-    },
-    "platform": "cocoa",
-    "exception": {
-        "values": [{
-            "value": "Attempted to dereference garbage pointer 0x1.",
-            "mechanism": {
-                "posix_signal": {
-                    "name": "SIGSEGV",
-                    "code_name": "SEGV_NOOP",
-                    "signal": 11,
-                    "code": 0
-                },
-                "relevant_address": "0x1",
-                "mach_exception": {
-                    "exception": 1,
-                    "exception_name": "EXC_BAD_ACCESS",
-                    "subcode": 8,
-                    "code": 1
-                }
+  "debug_meta" : {
+    "images" : [
+      {
+        "name" : "CrashProbeiOS",
+        "image_vmaddr" : "0x0000000100000000",
+        "image_addr" : "0x0000000100088000",
```

Comment:
Is there a way to shorten these? With compression, the benefit is very limited, but it might still make it more readable.

Reply:
I would rather not tinker with this now, it also is a PITA to update all tests that depend on that.